eagle-server/src/main/webapp/app/dev/public/js/ctrls/alertEditCtrl.js

```diff
@@ -304,6 +304,15 @@
 			return false;
 		};
 
+
+		$scope.checkPolicyName = function () {
+			if($scope.policy.name.length > 50) {
+				return "length should less than 50";
```

Comment:
should we say "policy name length should be less than 50" ?  just length sounds very generic.